.dockstore.yml

```diff
@@ -11,7 +11,7 @@ workflows:
 
   - name: ArrayImputationQC
     subclass: WDL
-    primaryDescriptorPath: /pipelines/broad/arrays/imputation_beagle/ArrayImputationQC.wdl
+    primaryDescriptorPath: /pipelines/wdl/arrays/imputation_beagle/input_qc/ArrayImputationQC.wdl
```

Comment:
hmm any idea why this needed to be updated but the rest of the workflows in this file seem to be fine?